lang/cs/Org.Apache.REEF.IO.Tests/TestHadoopFileSystem.cs

```diff
@@ -142,6 +143,27 @@ public void TestCreate()
             Assert.Throws<NotImplementedException>(() => _fileSystem.Create(GetTempUri()));
         }
 
+        [Fact(Skip = "These tests need to be run in an environment with HDFS installed.")]
+        public void CreateUriForPathNoPrefix()
+        {
+            Uri uri = _fileSystem.CreateUriForPath("/tmp/TestHadoop");
+            Assert.True(new Regex("hdfs://[a-z]+:\\d+/tmp/TestHadoop").Match(uri.AbsoluteUri).Success);
+        }
+
+        [Fact(Skip = "These tests need to be run in an environment with HDFS installed.")]
+        public void TestCreateUriForPathInvalid()
+        {
+            Assert.Throws<ArgumentException>(() => _fileSystem.CreateUriForPath("http://www.invalidhadoopfs.com/container/folder1/file1.txt"));
+        }
+
+        [Fact(Skip = "These tests need to be run in an environment with HDFS installed.")]
+        public void CreateUriForPathWithPrefix()
+        {
+            string uriString = "hdfs://localhost:9000/tmp/TestHadoop";
```

Comment:
supernit: try to limit swap variables when possible.
```csharp
var uri = new Uri("hdfs://localhost:9000/tmp/TestHadoop");
Assert.Equal(uri, _fileSystem.CreateUriForPath(uri.ToString()));
```

Reply:
Removed uri variable.